lib/srv/sess.go

```diff
@@ -718,7 +718,7 @@ type session struct {
 }
 
 // newSession creates a new session with a given ID within a given context.
-func newSession(ctx context.Context, id rsession.ID, r *SessionRegistry, scx *ServerContext, ch ssh.Channel) (*session, *party, error) {
+func newSession(ctx context.Context, id rsession.ID, r *SessionRegistry, scx *ServerContext, ch ssh.Channel, interactive bool) (*session, *party, error) {
```

Comment:
💅 We could create a new method here `newInteractiveSession` instead of adding a boolean.

Reply:
I didn't meant to suggest duplicating the logic
Just the entry point: `newTrackedSession/newInteractiveSession` and `newUntrackedSession` (or similar)
And then pull all the logic into a new fn `newBaseSession(..., trackSession bool) ...` (with a godoc indicating that only the above methods should be used.

This is a private method, so we can be more relaxed about it. However, seeing all those `true/false` arguments is somewhat confusing.